### PR TITLE
Update Polish (pl) translation

### PR DESCRIPTION
**Pull request by FeralChild64**

### CODE REVIEW

**Review by dreamer — COMMENTED** [review]
I noticed several typos, perhaps you fix them while updating the translation :)

File: contrib/resources/translations/utf-8/pl.txt [hunk 1]
```diff
@@ -885,18 +903,19 @@ Patrz 'serial1'
 Patrz 'serial1'
 .
 :CONFIG_PHONEBOOKFILE
-Plik z informacją o mapowaniu wirtualnych numerów telefonicznych na adresy.
+Plik z informacją o mapowaniu wirtualnych numerów telefonicznych na adresy
+(domyslnie 'phonebook.txt').
```
[the remark this answers is not in the document]
Comment (FeralChild64) [reply contrib/resources/translations/utf-8/pl.txt]: Fixed. Nice to have a reviewer for such things :)

File: contrib/resources/translations/utf-8/pl.txt [hunk 2]
```diff
@@ -906,19 +925,20 @@ Typowe wartości to: 3.3, 5.0, 6.22, oraz 7.1.
...
-'pl214' (polski układ maszynistki).
+Kod języka klawiatury (domyślnie 'auto') - np. 'pl' (polski układ
+programisty), 'pl214' (polski układ maszynistki).
 .
 :CONFIG_IPX
-Włącza emulację protokołu IPX po UDP/IP.
+Włącza emulację protokołu IPX po UDP/IP (domyślnie włączone).
 .
 :CONFIG_NE2000
 Włącza emulację karty sieciowej Novell NE2000 podłączonej do wirtualnej
```
[the remark this answers is not in the document]
Comment (FeralChild64) [reply contrib/resources/translations/utf-8/pl.txt:944]: Fixed.

File: contrib/resources/translations/utf-8/pl.txt [hunk 3]
```diff
@@ -928,26 +948,26 @@ sieci 'libslirp' z parametrami jak poniżej:
...
 :CONFIG_NICBASE
-Adres bazowy (I/O) karty sieciowej NE2000.
+Adres bazowy (I/O) karty sieciowej NE2000 (domyślnie 300).
 Uwaga: Adresy 220 oraz 240 ogą być niedostępne, gdyż są domyślnie przypisane
        do kart muzycznych Sound Blaster oraz Gravis UltraSound.
 .
 :CONFIG_NICIRQ
-Numer przerwania (IRQ) karty sieciowej NE2000.
-Uwaga: Przerwania 3 oraz 5 mogą być niedostępne, gdyż są domyślnie przypisane
+Numer przerwania (IRQ) karty sieciowej NE2000 (domyslnie 3).
```
[the remark this answers is not in the document]
Comment (FeralChild64) [reply contrib/resources/translations/utf-8/pl.txt]: Fixed.

**Pull request merged**